debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -1093,6 +1093,16 @@ public static final Field MASK_COLUMN(int length) {
                             + "be masked with " + length + " asterisk ('*') characters.");
     }
 
+    public static final Field ENABLE_TIME_ADJUSTER = Field.create("enable.time.adjuster")
+            .withDisplayName("Enable Time Adjuster")
+            .withType(Type.BOOLEAN)
+            .withDefault(false)
+            .withWidth(Width.SHORT)
+            .withImportance(Importance.LOW)
+            .withDescription("MySQL allows user to insert year value as either 2-digit or 4-digit. In case of two digit the value is automatically mapped into 1970 - 2069." +
+                    "false - (the default) delegates the implicit conversion to the database" +
+                    "true - Debezium makes the conversion");
```

Comment:
How will that conversion be made in this case?

Reply:
The conversion is done by `MySqlValueConverters::adjustTemporal`, see https://github.com/debezium/debezium/pull/829/files#diff-b973c03eead4ea581c3008fd911c7536R150